content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/amp.html

```diff
@@ -0,0 +1,53 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+  
+<div data-sly-use.tabs="com.adobe.cq.wcm.core.components.models.Tabs"
```

Comment:
@gaetanmarmasse - I think for the tabs AMP component we may need to add some logic that ensures that if no active item is explicitly set by an author, the first tab should be selected. This would be in line with how the existing tabs component behaves.

Reply:
I believe this is how it works today in this component. Did you notice it was not?

Reply:
Tested this again, attaching an example - I would expect the first tab to be selected if no selection was explicitly defined by an author.
![tabs-no-default-active-item](https://user-images.githubusercontent.com/25616660/73167901-0699d280-40f9-11ea-9306-2b2888cea5e4.gif)

Reply:
@gaetanmarmasse - created a separate pull request for this, that ensures the first item is marked active if no active item was configured: https://github.com/Bounteous-Inc/aem-core-wcm-components/pull/55